recipe/meta.yaml

```
@@ -22,14 +25,80 @@ requirements:
   host:
     - ncurses
 
-test:
-  commands:
-    - test -f $PREFIX/lib/libestbase.so  # [linux]
-    - test -f $PREFIX/lib/libestbase.so.2.5  # [linux]
-    - test -f $PREFIX/lib/libestools.so  # [linux]
-    - test -f $PREFIX/lib/libestools.so.2.5  # [linux]
-    - test -f $PREFIX/lib/libeststring.so  # [linux]
-    - test -f $PREFIX/lib/libeststring.so.2.5  # [linux]
+outputs:
+  - name: libestools
+    script: install.sh  # [linux]
+    build:
+      run_exports:
+        - {{ pin_subpackage('libestools', max_pin='x.x') }}
+
```

```suggestion

    requirements:
      build:
        - {{ compiler('c') }}
        - {{ compiler('cxx') }}
        - libgomp  # [linux]
      host:
        - ncurses

```
Need to add these dependencies since toplevel package is now named libestools-split.